README.md

```diff
@@ -2,9 +2,22 @@
 
 Sistema de gerenciamento de estoques, lojas e incentivo ao colaborador
 
-### Como rodar?
+### Como rodar o projeto?
 
 1. Abra a pasta `\estoquei-project` no terminal;
 2. digite `npm install` para instalar as dependências;
 3. Rode o servidor com o comando `npx json-server --watch src/assets/data/database.json`;
 4. Rode o front com o comando `npm start`.
+
+
+### Como rodar os testes?
```

Comment:
I think that it's more about how to run our project. What do you think about?

```suggestion
```

Reply:
In fact, the could have been just one point. I just split them to clarify that when you're going to run tests, the project should be up. It wasn't very clear to me when i first runt he project and needed to run the tests. So i will keep them separated but on ""Como rodar os testes?"" i wont repeat the same text, just reference it.

Reply:
Nice, thank you by your feedback!

Reply:
@marcoafsantana Feel free to merge your PR.